nRFBlinky/ViewControllers/ScannerView/ScannerTableViewController.swift

```diff
@@ -102,10 +106,13 @@ class ScannerTableViewController: UITableViewController, CBCentralManagerDelegat
         if !discoveredPeripherals.contains(newPeripheral) {
             discoveredPeripherals.append(newPeripheral)
             tableView.beginUpdates()
+            if discoveredPeripherals.count == 1 {
+                tableView.insertSections(IndexSet(integer: 0), with: .fade)
+            }
             tableView.insertRows(at: [IndexPath(row: discoveredPeripherals.count - 1, section: 0)], with: .fade)
             tableView.endUpdates()
         } else {
-            if let index = discoveredPeripherals.index(of: newPeripheral) {
```

Comment:
(save state_discovered : savePeripheral(s))()= survive always supervisor; root check and fix "index" as indexJ;*}  save state save_all ON